### DISCUSSION TITLE
Missing Build Artifact for macOS - Intel

### DISCUSSION THREAD

**Discussion by anitconsultantllc** [jgraph/drawio-desktop#676] [General]
https://github.com/jgraph/drawio-desktop/releases/tag/v14.9.6, "macOS - Intel", artifact doesn't exist and isn't being generated by the build process or at least the link is bad.

<img width="1633" alt="Screen Shot 2021-08-18 at 5 59 19 PM" src="https://user-images.githubusercontent.com/6926175/129991224-463ba2e8-34f8-49c0-aa45-389ef7b2ea71.png">

**Comment by davidjgraph** [comment]
Fixed, thanks for report.
